lib/api.js

```diff
@@ -22,22 +22,33 @@ const BASE_URL = 'https://api.voxel51.com/v1';
  * Main class for managing a session with the Voxel51 Vision Services API.
  */
 class API {
+
```

Comment:
Reason for the extra line here? Just consistent formatting?

Reply:
yes just consistent formatting.